example-app/src/main/res/values/strings.xml

```diff
@@ -44,6 +44,8 @@
     <string name="split_card_funding_sources_key">split_card_funding_sources</string>
     <string name="split_card_funding_sources_title">Split Card Funding Sources</string>
     <string name="enable_card_address_form_key">enable_card_address_form</string>
+    <string name="enable_card_installment_options_title">Card installment options type</string>
+    <string name="enable_card_installment_options_key">enable_card_installment_options_key</string>
```

Comment:
This reads like a boolean now but it actually can have 4 values right?

Reply:
True and the reason for that because it's enable the installment option with the 3 options or disable it so I didn't know what else to call it WDYT?

Reply:
Maybe something like `mode` instead of `enable` to make it look like an enum not boolean?

Reply:
Make sure you use the same naming everywhere, there's still stuff that start with `enable_card_installment_options_...` or `settingsInstallmentOptionType` that are not consistent with the rest